### PR TITLE
Issue/592 agpl to gpl (Connect #592)

### PR DESCRIPTION
**Pull request by stellanl**
#### Before the PR (what is the issue or what needed to be done)
Most files had AGPL headers, a few had none or headers unchanged from Eclipse templates.
#### The solution
Changed all headers to GPL3, changed name in copyright notice from FLOW to Flow, added headers where missing, made formatting more consistent.
#### Reviewer Checklist
* [ ] Connect the issue
* [ ] Copyright header
* [ ] Code formatting

### CODE REVIEW

File: app/src/main/java/org/akvo/flow/activity/AddUserActivity.java [hunk 1]
```diff
@@ -1,17 +1,20 @@
 /*
  *  Copyright (C) 2015-2016 Stichting Akvo (Akvo Foundation)
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/activity/AddUserActivity.java]: Should be 2017

File: app/src/main/java/org/akvo/flow/activity/AppUpdateActivity.java [hunk 2]
```diff
@@ -1,17 +1,20 @@
 /*
  *  Copyright (C) 2014-2016 Stichting Akvo (Akvo Foundation)
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/activity/AppUpdateActivity.java]: Should be 2017

File: app/src/main/java/org/akvo/flow/activity/BackActivity.java [hunk 3]
```diff
@@ -1,18 +1,20 @@
 /*
- * Copyright (C) 2010-2016 Stichting Akvo (Akvo Foundation)
+ *  Copyright (C) 2010-2016 Stichting Akvo (Akvo Foundation)
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/activity/BackActivity.java]: Should be 2017

File: app/src/main/java/org/akvo/flow/activity/GeoshapeActivity.java [hunk 4]
```diff
@@ -1,18 +1,22 @@
 /*
  *  Copyright (C) 2015 Stichting Akvo (Akvo Foundation)
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/activity/GeoshapeActivity.java]: update the year, should it be 2015-2017?

**Comment by muloem** [comment]
@valllllll2000 @stellanl FYI the copyright year should only be updated in case that source code in that file was actually modified that year. so if there are files that were modified in 2017 update them otherwise the Copyright year (range) should remain as is.

**Comment by valllllll2000** [comment]
@muloem of course but in this case, he modified the copyright header of all the files so the files were modified so it should be updated, right? Or does the copyright notice not count as modification?

**Comment by muloem** [comment]
Given that we are copyrighting IP, I wouldn't count editing the copyright header as a modification.

**Comment by valllllll2000** [comment]
@stellanl @muloem ok I will remove my comments then and review the PR again

File: app/src/main/java/org/akvo/flow/api/S3Api.java [hunk 5]
```diff
@@ -1,3 +1,22 @@
+/*
+ *  Copyright (C) 2013 Stichting Akvo (Akvo Foundation)
```
Comment (valllllll2000) [inline app/src/main/java/org/akvo/flow/api/S3Api.java]: Why does it say 2013 here? If you just added this copyright notice, should it be 2017? or 2010-2017

**Comment by stellanl** [comment]
I did not touch the copyright year in the headers at all

**Pull request merged**